EVEStandard/SSO.cs

```diff
@@ -80,6 +80,19 @@ public Authorization AuthorizeToEVEURI(List<string> scopes)
             return this.AuthorizeToEVEURI(scopes, Convert.ToBase64String(Guid.NewGuid().ToByteArray()));
         }
 
+        /// <summary>
+        /// Generates the URL you should have users click on with one of the EVE Online provided button images.
+        /// A state parameter in the URL is optional, but expected for security so this function creates one based on a Base64 encoded Guid.
+        /// This is for Implicit Flow, where a secret key isn't sent
+        /// </summary>
+        /// <param name="scopes">List of the required scopes for your application</param>
+        /// <returns>The <c>Authorization</c> POCO with the SignInURI parameter set and the ExpectedState parameter set.</returns>
+        /// <exception cref="EVEStandardException" ><paramref name="scopes"/> parameter was empty or null</exception>
+        public Authorization AuthorizeToEVEURI_Implicit(List<string> scopes)
```

Comment:
Where these changes tested? Unless all the APIs have added so far I have testing the SSO class and so I'd like to keep it clean if possible. I also wondering if we should do either one of two things. Have ``` AuthorizeToEVEURI_Implicit ``` but then rename ``` AuthorizeToEVEURI ``` to  ```AuthorizeToEVEURI_Explicit ``` Or, only have ``` AuthorizeToEVEURI ``` and then just add ``` bool isImplicitFlow ``` to the function definition. You can then add that bool to the function on line 104, update the comments to explain how implicit workflow works and change lines 116 and 117 to use the bool to form whichever SignInURI we actually need for each workflow.

Also, could you tell me the steps that someone would take to utilize the implicit workflow after the callback?